web/libs/editor/src/tags/object/Video/Video.js

```diff
@@ -184,28 +187,40 @@ const Model = types
     },
 
     handleSync(data) {
+      if (data.initiator === self.name) return;
       if (!self.ref.current) return;
+      self.isProcessingIncomingSync = true;
```

Comment:
This should also be theoretically handled by mixin but this was never tested on TimeSeries of course. Which issues are there?